tests/docker-compose.auth-server.yml

```diff
@@ -0,0 +1,47 @@
+version: '3.3'
+
```

Comment:
please move the file into `tests/python/social_auth/` directory. All code for testing the feature should be in one place.